osquery/tables/yara/tests/yara_tests.cpp

```diff
@@ -80,4 +78,32 @@ TEST_F(YARATest, test_match_false) {
   // Should have 0 count
   EXPECT_TRUE(r["count"] == "0");
 }
+
+TEST_F(YARATest, should_skip_file) {
+  // pretty much any regular file should be scanned
+
+  EXPECT_FALSE(yaraShouldSkipFile("/any/file/here", S_IFREG));
+
+  // should skip osquery db LOCK
+
+  EXPECT_TRUE(yaraShouldSkipFile("/var/osquery/osquery.db/LOCK", S_IFREG));
```

Comment:
The default location of the DB should be in a utility library, `utils/config/default_path.h`.